vicar/vicarimage.py

```diff
@@ -714,385 +833,565 @@ def _read_file(filepath, extraneous='ignore'):
 
-            name_blanks: number of blank characters after the name and before the
-            equal sign; zero is the default.
+            where:
 
-            val_blanks: number of blank characters after the equal sign and before
-            the value; zero is the default.
+                * `format` is a format string, e.g., "%+7d" or "%7.3f".
+                * `name_blanks` is the number of blank characters after the name and
+                  before the equal sign; zero by default.
+                * `val_blanks` is the number of blank characters after the equal sign and
+                  before the value; zero by default.
+                * `sep_blanks` is the number of blanks after the value and before the next
+                  label parameter or the label's end; two by default.
 
-            sep_blanks: number of blanks after the value and before the next label
-            parameter or the label's end; a default value of zero means that the
-            standard padding (two blanks) will be used when the text string is
-            generated.
+            Note the use of square brackets in the tuple expression above. If the first
+            hint value is a string, it is interpreted as `format`; otherwise, the `format`
+            is unspecified. After the optional format, values are interpreted as numbers
+            of blanks. If only one int is provided, it defines `sep_blanks`, with
+            `val_blanks` and `name_blanks` set to zero. If two trailing ints are provided,
+            they define `val_blanks` and `sep_blanks`, with `name_blanks` set to zero.
 
-        If the value is a list, then each item in the list must be either a
-        parameter value (int, float, or string) or else a tuple of up to four
-        values:
+            For example, if the name is "TEXP" and the value is 1.5, this is how hint
+            values are interpreted::
 
-            (value[, format][[, blanks_before], blanks_after])
+                <no hints>        = ("", 0, 0, 0)     -> "TEXP=1.5  "
+                "%.3f"            = ("%.3f", 0, 0, 0) -> "TEXP=1.500  "
+                ("%.3f", 4)       = ("%.3f", 0, 0, 4) -> "TEXP=1.500    "
+                ("%.3f", 1, 4)    = ("%.3f", 0, 1, 4) -> "TEXP= 1.500    "
+                ("%.3f", 2, 1, 4) = ("%.3f", 2, 1, 4) -> "TEXP  = 1.500    "
+                4                 = ("", 0, 0, 4)     -> "TEXP=1.5    "
+                (1, 4)            = ("", 0, 1, 4)     -> "TEXP= 1.5    "
+                (2, 1, 4)         = ("", 2, 1, 4)     -> "TEXP  = 1.5    "
 
-        After the value, the optional items are:
+            When the parameter value is a list, it is also possible to embed formatting
+            information on an item by item basis. Replace any item value by a tuple:
 
-            format: a format string, e.g., "%+07d", "%12.3e" or "%.4f".
+                (`item`[, `format`][[, `blanks_before`], `blanks_after`])
```

Comment:
Some of these back-ticks are showing up in the HTML

Reply:
You can't put the second backtick just before a left bracket, apparently. I needed to insert a space before the bracket.